common/src/main/java/nearsoft/skytouch/common/config/RabbitMQConfigProperties.java

```diff
@@ -0,0 +1,93 @@
+package nearsoft.skytouch.common.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+@Component
+@Configuration
```

Comment:
Why Configuration?

Reply:
Because it is for the RabbitMQ configuration but i just realized that i'm using @Component too, i will use just @Configuration